rust/main/agents/relayer/src/msg/message_processor/disposition.rs

```diff
@@ -1,33 +1,42 @@
 use std::sync::Arc;
 
+use tracing::warn;
+
 use hyperlane_base::db::HyperlaneDb;
 use hyperlane_core::QueueOperation;
-use lander::Entrypoint;
+use lander::{Entrypoint, PayloadStatus, TransactionStatus};
 
 /// Disposition for how to handle an operation during submission check
 pub enum OperationDisposition {
```

Comment:
I think the naming of these variants are a bit confusing. Would be good to add to the doc that this enum is used in different stages.

Reply:
I have updated the doc comments for the enum and its variants. Please, let me know if it is still unclear. 

Any ideas how we can name the enum and its variants better?